tests/tests.toml

```
@@ -4138,3 +4138,7 @@ run = true
 [[test]]
 filename = "exit2.f90"
 run = true
+
+[[test]]
+name = "test_duplicate_module_procedures"
+path = "errors/test_duplicate_module_procedures.f90"
```

Please change that to 
```suggestion

[[test]]
filename = "errors/test_duplicate_module_procedures.f90"
run = true
```
then run `run_tests.py  -u`, then push that commit.

---

with correct fix it should raise an error like this one ` Entity ‘test_01’ at (1) is already present in the interface`